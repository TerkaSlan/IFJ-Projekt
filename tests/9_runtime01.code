/*Division by zero*/

class Main {

	static void run() {
		int zero = 0;
		int result;
		result = 10 / zero;
	}
}