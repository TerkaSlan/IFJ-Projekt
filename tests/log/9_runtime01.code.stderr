ERROR: Runtime - division by zero - Trying to divide by 0. Operand zero
